tests/unit/api/test_tasks.py

```diff
@@ -75,7 +75,7 @@ def test_get_invalid_scope(self, test_client, monkeypatch):
         token_payload = {
             "username": "admin",
             "password": "secret",
-            "scope": ("read:targets " "read:settings " "read:token "),
+            "scope": ("read:settings"),
```

Comment:
```suggestion
            "scope": "read:settings",
```
The scope is a string